doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "MultiTech Conduit AEP"
+description: ""
+weight: 8
+---
+
+## Connecting MultiTech Conduit to {{%tts%}}
+
+The [MultiConnect® Conduit™](http://www.multitech.net/developer/products/multiconnect-conduit-platform/) is a configurable, scalable cellular communications gateway for industrial IoT applications. The technical specifications of this conduit can be found in the **[official documentation](https://www.multitech.com/documents/publications/software-guides/s000727%20-%20mPower%20Edge%20Intelligence%20Conduit_AEP_software_guide.pdf).**
+
+This guide will help you set up the gateway to communicate over **{{%tts%}}**.
+
+{{< figure src="001_Multitech_Conduit_AEP.png" alt="multitech Conduit AEP" >}}
+
+<!--more-->
+
+<br>
+### Prerequisites
+
+-   MultiTech Conduit [AEP](http://www.multitech.net/developer/software/aep/) model. 
+-   MultiTech [MTAC-LoRa LoRa accessory card](http://www.multitech.net/developer/products/accessory-cards/mtac-lora/), installed as [instructed](http://www.multitech.net/developer/products/accessory-cards/installing-an-accessory-card/).
+Do not forget to mount the antenna to the mCard after fitting it in the conduit.
+-   Computer with an Ethernet port.
+-   LoRa Antenna - 1
+-   LTE Antennas - 2
+-   WiFi Antenna - 1
+-   LAN Cable - 1
+-   GPS Antenna - 1
+-   Screwdriver
+-   SIM Card - 1 (If you use cellular as backhaul)
+<br>
+{{< figure src="002_Hardware_Setup_Parts.png" alt="Hardware Setup Parts" >}}
+
+<br>
+## Hardware Setup:
+
+- Attach the LoRa antenna to LoRa mCards’s RF connector in the AP2 slot located at the back of the conduit. Finger-tighten the antenna. (Ignore this step, if the antenna is already attached.)
+- Similarly, attach GPS, WiFi, CELL and LoRa antennas.
+- Attach the power supply to the Multitech Conduit.
+
+<br>
+{{< figure src="003_Multitech_Conduit_Ports.png" alt="Hardware Setup Parts" >}}
+<br>
+{{< figure src="004_Antenna_Assembled.png" alt="004_Antenna_Assembled.png" >}}
+
+<br>
+
+## Configuring the AEP Model
+
+### Login to the Web Interface
+
+To start configuring your conduit, connect a computer to its LAN interface and open the following URL in a browser: [http://192.168.2.1/](http://192.168.2.1/).  The URL redirects you to the commissioning window to set a Username and Password.
```

Comment:
- Conduit with uppercase (it's a name)
- No need to have multiple spaces after the period. Also in other places.
- username and password can be lowercase